backend/src/index.js

```diff
@@ -0,0 +1,58 @@
+const autobahn = require("autobahn");
+
+const models = require("../models");
+
+const wamp = new autobahn.Connection({
+  realm: "near-explorer",
+  transports: [
+    {
+      url: process.env.WAMP_NEAR_EXPLORER_URL || "ws://localhost:8080/ws",
+      type: "websocket"
+    }
+  ],
+  authmethods: ["ticket"],
+  authid: "near-explorer-backend",
+  onchallenge: (session, method, extra) => {
+    if (method === "ticket") {
+      return process.env.WAMP_NEAR_EXPLORER_BACKEND_SECRET || "back";
+    }
+    throw "WAMP authentication error: unsupported challenge method";
+  }
+});
+
+async function main() {
+  wamp.onopen = async session => {
+    console.log("WAMP connection is established. Waiting for commands...");
+    await session.register(
```

Comment:
This is where the backend handles the requests (there is an example of a message broadcast below)